conans/test/unittests/util/tools_test.py

```diff
@@ -144,6 +144,19 @@ def cpu_count_test(self):
             with six.assertRaisesRegex(self, ConanException, "Invalid CONAN_CPU_COUNT value"):
                 tools.cpu_count(output=output)
 
+    def test_cpu_count_in_container(self):
```

Comment:
This test is not working. The idea here is patching tools.load to return a specific value. Any ideas how to fix it?

Reply:
maybe because load has two args?

Reply:
no, I think it's related to path used in patch, but I can't figure out.

Reply:
It's working now. I just exported helper functions to a new class.